fabric-lifecycle-events-v1/build.gradle

```diff
@@ -1,5 +1,5 @@
 archivesBaseName = "fabric-lifecycle-events-v1"
-version = getSubprojectVersion(project, "1.0.0")
+version = getSubprojectVersion(project, "1.0.1")
```

Comment:
should bump the minor instead of the patch

Reply:
done